tests/src/test/scala/com/nvidia/spark/rapids/UrlFunctionsSuite.scala

```diff
@@ -0,0 +1,177 @@
+/*
+ * Copyright (c) 2023, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.nvidia.spark.rapids
+
+import org.apache.spark.sql.{DataFrame, SparkSession}
+
+class UrlFunctionsSuite extends SparkQueryCompareTestSuite {
```

Comment:
Should this just be moved into the pyspark integration test?  I don't see anything this test is doing that cannot be done in the integration test, and the latter is preferable since it executes in parallel.

Reply:
Ok, moved to integration test.